packages/common/src/util.ts

```diff
@@ -115,6 +115,46 @@ export function createId(prefix: string = "i"): string {
     .substr(2, 8)}`;
 }
 
+/**
+ * If value is not null, push it into an array, or append as a property of an object
+ * Allows for code like:
+ * ```js
+ * const vals = maybeAdd([], getProp(obj, 'some.deep.path.thatMayBe.undefined'));
+ * ```
```

Comment:
i'm _pretty sure_ our static site generator can only deal with a single snippet though, so your TypeDoc might make it barf. (you can run `npm run doc:serve` to find out for sure.)

another option would be to just rip off the _flawless_ ✨ example here:
https://github.com/Esri/arcgis-rest-js/blob/8173aa5c2d92b2f3f47ff108a8fb575fe449b99b/packages/arcgis-rest-geocoder/src/geocode.ts#L68-L87